src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/tests/Table.test.js

```diff
@@ -371,6 +390,59 @@ test('Clicking a checkbox should call onRowSelectionChange with the selection st
     expect(onChangeSpy).toHaveBeenCalledWith(rowIdOne, true);
 });
 
+test('Select-all checkbox should be checked if every non-disabled line is selected', () => {
```

Comment:
Any reason not to split this test into three different tests?

Reply:
done 🙂